website/blog/2023-05-26-7.22.0.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title:  "7.22.0 Released: Explicit Resource Management support and Import Attributes parsing"
+author: Babel Team
+authorURL: https://twitter.com/babeljs
+date:   2022-05-26 0:00:00
+categories: announcements
+share_text: "Babel 7.22.0 Released"
+---
+
+Babel 7.22.0 is out, with parsing/transform support for the [Explicit Resource Management](https://github.com/tc39/proposal-explicit-resource-management) proposal, including both the sync and async variants, and with parsing support for the [Import Attributes](https://github.com/tc39/proposal-import-attributes) (an evolution of the old [Import Assertions](http://web.archive.org/web/20220930084518/https://github.com/tc39/proposal-import-assertions) proposal).
```

Comment:
Would https://github.com/tc39/proposal-import-attributes/tree/69183bda2ecbac7f7ed71d961d8d8820c56e3c0b be better?

Reply:
I prefer the webarchive URL just because it says "proposal-import-assertions" in the repo name :)